### PR TITLE
Glift

### PR DESCRIPTION
**Pull request by alfbr**
Added a case with well gas lift to complement the group gas lift.

### CODE REVIEW

**Comment by alfbr** [comment]
jenkins build this please

**Comment by tskille** [comment]
Thanks Alf.  I believe the order of the test cases should be

0_BASE_MODEL5.DATA  -  The most basic model
1_MAXVALUE_TRANZ.DATA  - very basic functionality (Edit section of the model)
2_WLIFT_MODEL5.DATA  - needs to before the network test model since lift gas needs to be included in network modelling
3_GLIFT_MODEL5.DATA 
4_NETWORK_MODEL5_STW.DAT

model 0, 1 and 2 should then be possible to include in regression testing. Model 3 and 4 needs changes before these can be included in regression testing.

File: model5/2_WLIFT_MODEL5.DATA [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+ 6000*10000.0 /
+
+PERMZ
+ 6000*1000.0 /
+
+
+COPY
+  PERMX PERMY /
+/
+
```
Comment (tskille) [inline model5/2_WLIFT_MODEL5.DATA:99]: Use same permeability model as in MAXVALUE_TRANZ sin this model is defined before 2_WLIFT_MODL5

```
INCLUDE
 'include/permx.inc' /

PORO
 6000*0.28 /

COPY
  PERMX PERMY /
  PERMX PERMZ /
/

MULTIPLY
 'PERMZ' 0.8 /
/
```

File: model5/2_WLIFT_MODEL5.DATA [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+EQUALS
+  'MULTY'  0.01 1 20  14 14  1 10 /
+/
+
+
+------------------------------------------------------------------------------------------------
+EDIT
+------------------------------------------------------------------------------------------------
+
```
Comment (tskille) [inline model5/2_WLIFT_MODEL5.DATA:112]: Also include MAXVALUE in EDIT section
```
-- actual maximum value is 816436 in this case
-- a max max value of 750000 should affect 76 cells
-- there are in other words 76 cells  

-- mean value without maxvalue tranz: 305174
-- mean value with maxvalue tranz: 304883


MAXVALUE
  TRANZ 750000 /
/
```

**Comment by alfbr** [comment]
Thanks for the review comments. The cases are now updated accordingly. Unfortunately there are issues. The base case shows deviations that needs further investigation and the WLIFT case with the TRAN modifers have convergence issues. I suggest we merge this as is, and sort out these issues in a later PR.

**Comment by tskille** [comment]
Excellent, merging now.

**Pull request merged**